Dockerfile.kubearmor

```diff
@@ -0,0 +1,31 @@
+### Builder
+
+FROM accuknox/kubearmor:base as builder
+
+WORKDIR /usr/src/KubeArmor
+
+COPY ./KubeArmor ./KubeArmor
+COPY ./GKE ./GKE
+COPY ./protobuf ./protobuf
+
+WORKDIR /usr/src/KubeArmor/KubeArmor
+
+RUN ./build/patch.sh
+RUN GOOS=linux GOARCH=amd64 go build -a -ldflags '-s -w' -o kubearmor main.go
+
+### Make executable image
+
+FROM alpine:3.12
+
+RUN apk update
+RUN echo "@edge http://dl-cdn.alpinelinux.org/alpine/edge/testing" | tee -a /etc/apk/repositories
+
+RUN apk update
+RUN apk add bash curl procps bcc bcc-dev
+RUN apk add apparmor@edge
```

Comment:
Before:
RUN apk add apparmor@edge

Need to change:
RUN apk add apparmor@edge apparmor-utils@edge